setup.py

```diff
@@ -33,11 +33,14 @@
               'dpdata/siesta', 
               'dpdata/gaussian', 
               'dpdata/cp2k',
+              'dpdata/ase',
               'dpdata/xyz',
               'dpdata/pwmat', 
               'dpdata/amber',
               'dpdata/fhi_aims',
-              'dpdata/gromacs'],
+              'dpdata/gromacs',
+              'dpdata/ase'
```

Comment:
You need to use another directory name instead of `ase`, otherwise it will be conflict with the `ase` package.